README.md

```diff
@@ -1,17 +1,23 @@
 # cloudland2023-otel-workshop
 
-A tutorial for monitoring and troubleshooting applications in production using OpenTelemetry.
+_Workshop content for a session during
+[Cloud Land 2023](https://www.cloudland.org/en/home/) -- The content may not be
+updated after the conference_
+
+A tutorial for monitoring and troubleshooting applications in production using OpenTelemetry (OTel).
 In this tutorial, you will create a simple express server, then create a `tracer.js` file that uses OpenTelemetry components and will produce traces from your application. In the last step, you will see your traces in Jaeger's backend.
 
 ## Table of Contents
 
-- [Prerequisites](#pre-requisites)
-- [Step 1 - Build an express app](#step-1---build-an-express-app)
-- [Step 2 - Create a tracer](#step-2---create-a-tracer)
-- [Step 3 - Use tracer.js in the app](#step-3---use-tracerjs-in-the-app)
-- [Step 4 - Do you have docker installed? Let's work locally](#step-4---do-you-have-docker-installed-lets-work-locally)
-- [Step 5 - Automatic instrumentations](#step-5---automatic-instrumentations)
-- [Step 6 - Create a span manually](#step-6---create-a-span-manually)
+- [cloudland2023-otel-workshop](#cloudland2023-otel-workshop)
+  - [Table of Contents](#table-of-contents)
+  - [Prerequisites](#prerequisites)
+  - [Step 1 - Build an express app](#step-1---build-an-express-app)
+  - [Step 2 - Create a tracer](#step-2---create-a-tracer)
+  - [Step 3 - use `tracer.js` in the app](#step-3---use-tracerjs-in-the-app)
+  - [Step 4 - Do you have docker installed? let's work locally](#step-4---do-you-have-docker-installed-lets-work-locally)
+  - [Step 5 - Automatic instrumentations](#step-5---automatic-instrumentations)
+  - [Step 6 - Create a span manually](#step-6---create-a-span-manually)
```

Comment:
My vscode made some auto updates to the TOC, feel free to ignore that one
```suggestion
- [Prerequisites](#pre-requisites)
- [Step 1 - Build an express app](#step-1---build-an-express-app)
- [Step 2 - Create a tracer](#step-2---create-a-tracer)
- [Step 3 - Use tracer.js in the app](#step-3---use-tracerjs-in-the-app)
- [Step 4 - Do you have docker installed? Let's work locally](#step-4---do-you-have-docker-installed-lets-work-locally)
- [Step 5 - Automatic instrumentations](#step-5---automatic-instrumentations)
- [Step 6 - Create a span manually](#step-6---create-a-span-manually)
```

Reply:
I didn't like when it put "Table of content" inside "Table of content" :) this is why I turned off this automatic TOC feature at my VScode.